### ISSUE TITLE
Fog Bias not working

### ISSUE THREAD

**Issue by cyberlecs** [bdancer/blender-for-vray#224]
Hi,
Fog bias seems not to be working in the last version. No matter what value I use, the result is always the same.

Tested with this nightlies builds:
V-Ray: vraystd_adv_36101_x64_vc101_hwlic_27448
Blender: vrayblender35-2.78-ae32ee9-x86_64-windows

Sample scene:
http://pasteall.org/blend/index.php?id=47290
![image_37791](https://user-images.githubusercontent.com/338739/28727167-448c1a8c-739a-11e7-937b-a4214feb328a.jpg)

**Comment by cyberlecs** [comment]
Sorry guys, my fault. 
​​​​​​​Fog bias is working fine (for some reason it is less sensible to values than it used to be).
![fog_bias2](https://user-images.githubusercontent.com/338739/28884477-6a02b4be-7788-11e7-80f0-0bf0de91204b.jpg)

**Comment by bdancer** [comment]
Yes, there was an internal multiplier previously, we should restore it.

**Comment by cyberlecs** [comment]
Hi guys, could you please take a look on this?
0.001 is the minimum value you can use in fog multiplier, but sometimes (even with very bright fog color) we would need to go lower than this to have a subtle fog effect (in scenes with 1unit=1meter).
Maybe restoring this internal multiplier or increasing one decimal digit in this field could work.
